tests/testData/userTestData.js

```diff
@@ -0,0 +1,26 @@
+const validUserLoginRaisa = {
+  username: "raisa",
+  password: "raisa",
+};
+
+const validUserLoginEthan = {
+  username: "ethan",
+  password: "123",
+};
+
+const invalidUserLoginEthan = {
+  username: "ethan",
+  password: "321",
+};
+
+const invalidUserLoginRaisa = {
+  username: "raisa",
+  password: "notraisa",
+};
+
+const pendingUserLoginAce = {
+  username: "ace",
+  password: "sparky",
+};
+
+export { validUserLoginRaisa, validUserLoginEthan, invalidUserLoginEthan, invalidUserLoginRaisa, pendingUserLoginAce };
```

Comment:
Instead of exporting all of these consts in one go at the end, I support putting the export keyword before each `const` declaration. That way, theoretically, people don't have to make additions in two different places to add more test data. That is just an opinion. If you have a justification for the current design that's okay, let me know.